File: muchos-motors/muchos-motors-api/Models/City.cs
using System.ComponentModel.DataAnnotations;

namespace muchos_motors_api.Models
{
    public class City
    {
        [Key]
        public int CityId { get; set; }
        public string Name { get; set; }
        public string PostCode { get; set; }
        public bool IsValid { get; set; }

    }
}

File: muchos-motors-front/src/app/shared/models/City.ts
import { OrderItem } from './OrderItem';

export interface ICity {
  cityId: number | undefined;
  name: string | undefined;
}

export class City implements ICity {
  _cityId: number | undefined;
  _name: string | undefined;

  get cityId() {
    return this._cityId;
  }

  get name() {
    return this._name;
  }
}
